src/main/java/org/terasology/dynamicCities/settlements/SettlementEntityManager.java

```diff
@@ -120,19 +121,21 @@
     private Random rng;
 
     private Logger logger = LoggerFactory.getLogger(SettlementEntityManager.class);
+
     @Override
     public void postBegin() {
 
         settlementEntities = settlementCachingSystem.getSettlementCacheEntity();
-        rng = new FastRandom(regionEntityManager.hashCode() & 0x921233);
+        long seed = regionEntityManager.hashCode() & 0x921233;
```

Comment:
Would it be worth trying to hook into the world seed instead? Also might help to make a variable/constant for the magic string here to explain its meaning (it seems fairly obvious but is a good practice)

Reply:
I think leaving this one as is is fine.
The names are hooked into the world seed below, this is for the other randoms, like city generation over time. If you hooked it into the seed it wouldn't really matter because all the other factors would be different